balancer-js/src/lib/utils/generate-address-books.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
```

Comment:
Is this file needed anywhere?

Reply:
Not really, this file is autogenerated by ts-node when the command "yarn address-book:generate" is executed

Reply:
Can we avoid commiting it somehow if its not required?